auditors/capabilities/fix.go

```diff
@@ -63,3 +62,24 @@ func removeCapabilityFromAddList(container *k8stypes.ContainerV1, capability str
 
 	container.SecurityContext.Capabilities.Add = added
 }
+
+type fixMissingSecurityContextOrCapability struct {
+	container *k8stypes.ContainerV1
+}
+
+func (f *fixMissingSecurityContextOrCapability) Plan() string {
+	return fmt.Sprintf("Adds capability to '%s' with drop set to ALL", f.container.Name)
+}
+
+func (f *fixMissingSecurityContextOrCapability) Apply(resource k8stypes.Resource) []k8stypes.Resource {
+	setDropToAll(f.container)
+	return nil
+}
+
+func setDropToAll(container *k8stypes.ContainerV1) {
+	container.SecurityContext = &k8stypes.SecurityContextV1{}
```

Comment:
If the container already has a security context I believe this will blow it away (we should only modify the capabilities)

Reply:
good catch!! I will change it!